app/uk/gov/hmrc/organisationsmatchingapi/models/Address.scala

```diff
@@ -18,7 +18,13 @@ package uk.gov.hmrc.organisationsmatchingapi.models
 
 import play.api.libs.json.Json
 
-case class Address(addressLine1: String, addressLine2: Option[String], addressLine3: Option[String], addressLine4: Option[String])
+case class Address(
```

Comment:
There will be a little work needed to merge the various domain models together.... a comment, rather than a change request! I think the matrix has these as "line1", "line2" etc, and "postcode" all lower case 🙂